mainnet/pirin-1/gentxs/nodesguru.json

```diff
@@ -0,0 +1,61 @@
+{
+  "body": {
+    "messages": [
+      {
+        "@type": "/cosmos.staking.v1beta1.MsgCreateValidator",
+        "description": {
+          "moniker": "Nodes.Guru",
+          "identity": "28B672FCE6BBD562",
+          "website": "https://stake.nodes.guru",
+          "security_contact": "security@nodes.guru",
+          "details": "Guru of non-custodial staking. Professional node running, low fees, best uptime and 24/7 customer support."
+        },
+        "commission": {
+          "rate": "0.050000000000000000",
+          "max_rate": "0.100000000000000000",
+          "max_change_rate": "0.050000000000000000"
+        },
+        "min_self_delegation": "1",
+        "delegator_address": "nolus1t2k2r4y9nl9cr5qa0xrzw8aaxz9dxl5p6nn3pa",
+        "validator_address": "nolusvaloper1t2k2r4y9nl9cr5qa0xrzw8aaxz9dxl5prrx5zq",
+        "pubkey": {
+          "@type": "/cosmos.crypto.ed25519.PubKey",
+          "key": "+XcUQxrcUGk2PyFQaY4ZeGJMV1FaZk564kzZcUqsRAg="
+        },
+        "value": {
+          "denom": "unls",
+          "amount": "5000000"
```

Comment:
wrong amount in self delegation should be `1000000unls`

Reply:
for the token allocation, not for the genesis stake
`nolusd add-genesis-account <wallet> 10000000unls `

token allocation will be done in the genesis by Nolus

Reply:
I suggest going with 1000000unls (1 NLS) for the self-delegation. There will be some tokens allocated to the address. What I meant with the comment is that it does not matter how much you give to this genesis account. You are overriding a default genesis.json that won't be the one for mainnet.